app/assets/javascripts/modules/mobile-filters-modal.js

```diff
@@ -108,34 +61,37 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     }
     // reset all selects, uncheck checkboxes, clear text input values
     // and remove the selected count on each facet
-    var $elements = this.$module.querySelectorAll('input, select, .js-selected-counter')
-    var $form = document.querySelector('.js-live-search-form')
+    var elements = this.module.querySelectorAll('input, select, .js-selected-counter')
+    var form = document.querySelector('.js-live-search-form')
     var customEvent = document.createEvent('HTMLEvents')
     customEvent.initEvent('customFormChange', true, false)
-    for (var i = 0; i < $elements.length; i++) {
-      var $el = $elements[i]
-      var tagName = $el.tagName
+    for (var i = 0; i < elements.length; i++) {
+      var el = elements[i]
+      var tagName = el.tagName
       switch (tagName) {
         case 'INPUT':
-          if ($el.type === 'checkbox' && $el.checked === true) {
-            $el.checked = false
-          } else if ($el.type === 'text' && $el.value !== '') {
-            $el.value = ''
+          if (el.type === 'checkbox' && el.checked === true) {
+            el.checked = false
+          } else if (el.type === 'text' && el.value !== '') {
+            el.value = ''
           }
           break
         case 'SELECT':
```

Comment:
Again not sure where to put this comment. On Chrome on the mobile view the contents of the select component appear very small and sometimes not apparently attached to the element itself. Not seeing the same thing in Firefox, so not sure what's going on, probably more to do with the component than this change. Something to investigate separately, perhaps.

![Screenshot 2023-02-21 at 15 39 35](https://user-images.githubusercontent.com/861310/220392633-6bf45370-f5a3-4684-b181-2973b6f2c491.png)

Reply:
@andysellick Are you using an external monitor? Mine looks like this 
<img width="456" alt="image" src="https://user-images.githubusercontent.com/8880610/221623013-20ef9553-7fe3-429c-984f-4f5e49e0e910.png">

Reply:
It happens when I switch to mobile view in Chrome but keep the window wide enough to still see the desktop site. Have raised an issue about it to avoid blocking this PR: https://github.com/alphagov/govuk_publishing_components/issues/3280